os/kernel/init/os_bringup.c

```diff
@@ -276,6 +276,10 @@ static inline void os_do_appstart(void)
 	}
 #endif
 
+#ifdef CONFIG_ERROR_REPORT
+	/* Initialize Error Reporting for network */
+	error_report_init();
```

Comment:
needs header to remove warning. `implicit-function-declaration`

Reply:
included the required header file. Thank you for suggesting.